webapp/src/main/java/org/apache/atlas/web/filters/AuditFilter.java

```diff
@@ -103,6 +103,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             requestContext.setCreateShellEntityForNonExistingReference(createShellEntityForNonExistingReference);
             requestContext.setForwardedAddresses(AtlasAuthorizationUtils.getForwardedAddressesFromRequest(httpRequest));
             requestContext.setSkipFailedEntities(skipFailedEntities);
+            requestContext.setClientOrigin(httpRequest.getHeader("X-Atlan-Client-Origin"));
```

Comment:
nit: "X-Atlan-Client-Origin" is a `static final constant`

Reply:
also handle when this header is not present. e.g. python SDK

Reply:
In case its not present it will not tag the requests, it is desired approach and it is not throwing NPE.

Reply:
@sumandas0 , It is not about NPE. If we decide to do metric and dashboard with this `origin`, it is better to use a default value rather than emptiness.

Reply:
Added